Dockerfile

```diff
@@ -1,11 +1,18 @@
+FROM golang:1.13 AS builder
```

Comment:
It would be nice to use our retagged golang image and 1.14.1 to match `architect`.

Reply:
Good thinking - updated